requirements.txt

```diff
@@ -1,3 +1,4 @@
 httpx
 pycryptodome>=3.4
 pyjwt>=2.1.0
+Pillow>=6.2.0
```

Comment:
Is there a reason for version pinning?

Reply:
Just import the latest version

Reply:
In that case, please remove the `>=6.2.0`.
The HA maintainers do have a look on these if we update the HA component and prefer as little package constraints as possible.